Pilot1/P1B2/p1b2_baseline_pytorch.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import print_function
+
+import numpy as np
+import os
+import sys
+
+import torch
+
+
+if True:
+    print("Restricting #of GPUs to 1")
+    os.environ["CUDA_VISIBLE_DEVICES"]="0"
+    #os.environ["CUDA_VISIBLE_DEVICES"]="0,1,2,3,4,5,6,7"
+    #os.environ["CUDA_VISIBLE_DEVICES"]="0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15"
```

Comment:
Please remove this. This environment variable should be set outside of this script.

Reply:
I will delete the environment variable. Thanks.